GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -186,11 +180,9 @@ public static bool TryCreateAzDevOrgUrlFromPackageFeedUrl(string packageFeedUrl,
         /// <param name="src">The unprocessed string to use as arguments to an install command</param>
         /// <param name="installationId">A unique installer ID to replace the installer_id token with.</param>
         /// <returns>The argument string with tokens replaced.</returns>
-        public static string ReplaceArgTokens(string src, string installationId)
+        public static string ReplaceArgTokens(string src, string installationId, string logsDirectory)

```

Comment:
Is this change still needed? Now that we're back to putting everything in the same directory will `logsDirectory` go back to always being `ProductUpgraderInfo.GetLogDirectoryPath()`?

Reply:
not technically needed - I thought it was a better fit, as now the caller is responsible for providing all the replacement tokens, rather than specifing 1, and having another one being handled automatically.

Reply:
Ok sounds good, thanks!